JSQMessagesViewController/Views/JSQMessagesCollectionView.m

```diff
@@ -37,6 +37,28 @@ - (void)jsq_configureCollectionView;
 
 @implementation JSQMessagesCollectionView
 
+- (void)setFrame:(CGRect)frame {
```

Comment:
JSQ repeatedly and unnecessarily changes the size of the collection view (and I think the input toolbar) while presenting/dismissing view controllers.  Conversation view needs to observe this so that it can make a best effort preserve scroll state.